src/tbp/monty/frameworks/config_utils/make_dataset_configs.py

```diff
@@ -1102,3 +1104,276 @@ class FiveLMMountConfig:
     zooms: List[float] = field(
         default_factory=lambda: [10.0, 10.0, 10.0, 10.0, 10.0, 1.0]
     )
+
+
+"""
+Utilities for generating arbitrary multi-LM dataset args.
+"""
+
+
+def make_multi_lm_sensor_positions(
```

Comment:
Again, a more descriptive name could be useful if we have other generator functions later. Maybe something that expresses that they are all attached to one agent and they evenly cover a surface? Also, should it maybe be `multi_sensor_positions`?

Reply:
How about `make_multi_sensor_positions_on_grid`?

I think `on_grid` expresses that they cover a surface, though I'm not sure they do evenly cover it when the ordering method is `distance`. Those arrangements can be kind of jagged on the outermost sensors.

I'm inclined to think `on_grid` also implies that they exist in the same agent space, but let me know if not.